WcaOnRails/db/migrate/20161031215259_create_currencies.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+class CreateCurrencies < ActiveRecord::Migration
+  def up
+    create_table :currencies do |t|
+      t.string :currency_code
+      t.string :description
+      t.string :currency_symbol
+      t.boolean :accepted_by_stripe
+      t.integer :display_divisor
+    end
+
+    execute <<-SQL
+      INSERT INTO currencies (
+        currency_code,
+        description,
+        currency_symbol,
+        accepted_by_stripe,
+        display_divisor
+      ) values
+      ('AUD','Australian Dollars','$', TRUE, 100),
```

Comment:
Lots of questions! Please bear with me.

Where did this data come from? Do we need to do database joins on any of this? If no, could we use a third party lib for this?

It seems strange to me that some of the `currency_symbol`s are html entities, and others are unicode characters. I'd vote for all unicode characters if possible.